### PR TITLE
Delete donate

### PR DESCRIPTION
**Pull request by wiliajc87**

### CODE REVIEW

**Comment by sowiecki** [comment]
"We can’t automatically merge this pull request." @wiliajc87

**Comment by Nase00** [comment]
:+:

File: config/routes.rb [hunk 1]
```diff
@@ -11,8 +11,13 @@
   get 'our_services' => 'welcome#our_services'
   get 'contact_us' => 'welcome#contact_us'
 
+<<<<<<< HEAD
```
Comment (sowiecki) [inline config/routes.rb]: Gotta remove this.

**Comment by sowiecki** [comment]
:+1:

**Comment by thewildpendulum** [comment]
@wiliajc87 Fix merge conflicts

**Pull request merged**